templates/CRM/CiviAwards/Form/AwardCustomField.tpl

```diff
@@ -1,6 +1,6 @@
-{if $invalidAwardType}
+{if $invalidAwardSubtype}
   <p class="alert alert-warning">
```

Comment:
No need to touch rename the variable. The error is related to case type not award sub types.
We can change the message maybe to reflect that?